tests/drivelist.spec.js

```diff
@@ -32,10 +32,27 @@ describe('Drivelist', function() {
         this.executeExtractAndRunStub = m.sinon.stub(execute, 'extractAndRun');
 
         this.executeExtractAndRunStub.withArgs(scripts.linux).yields(null, [
-          'device: "/dev/sda"',
-          'description: "My drive"',
-          'size: "15 GB"',
-          'mountpoint: "/mnt/drive"'
+          'enumerator: lsblk',
+          'busType: UNKNOWN',
```

Comment:
I guess it doesn't matter, but these `lsblk` and `UNKNOWN` string values should probably be quoted? (like all the other string key-values here)